proto/shentu/oracle/v1alpha1/oracle.proto

```diff
@@ -119,3 +119,11 @@ message TxTask {
     int64 score = 7 [(gogoproto.moretags) = "yaml:\"score\""];
     TaskStatus status = 8 [(gogoproto.moretags) = "yaml:\"status\""];
 }
+
+message LeftBounty {
```

Comment:
I think RemainingBounty is a bit better, any other ideas?

Reply:
remaining bounty is better